mqtt/mqttd/src/broker/bootstrap/edgehub.rs

```diff
@@ -150,10 +152,12 @@ where
         broker_ready.send(()).expect("ready signal");
     });
 
+    let device_id = env::var(DEVICE_ID_ENV).context(DEVICE_ID_ENV)?;
```

Comment:
@and-rewsmith, @dmolokanov, @vipeller  I think I've asked before, but I forgot... do we really need to have "device_id" in the client_id for the CommandHandler?

Reply:
I've made this tiny refactoring for proper error message, but I'm happy to remove this completely if we don't need this.